overlord/devicestate/devicemgr.go

```diff
@@ -875,6 +889,11 @@ func (m *DeviceManager) RequestSystemAction(systemLabel string, action SystemAct
 		return ErrUnsupportedAction
 	}
 
+	if sysAction.Mode == "run" {
```

Comment:
this probably needs a comment that we assume for now we are in run mode,  later when we let this be invoked also through tools etc,  we might be in recover mode in which all of this will need to behave a bit differently